apps/backoffice-v2/entrypoint.sh

```
@@ -0,0 +1,53 @@
+#!/usr/bin/env sh
+
+if [[ -n "$VITE_DOMAIN" ]]
```

Shellcheck (0.10.0)

[warning] In POSIX sh, [[ ]] is undefined.

(SC3010)